bokeh/embed/util.py

```diff
@@ -160,7 +160,7 @@ def __init__(self, docid=None, sessionid=None, elementid=None, roots=None, use_f
         if roots is None:
             roots = OrderedDict()
         elif isinstance(roots, list):
-            roots = OrderedDict([ (root, make_id()) for root in roots ])
+            roots = OrderedDict([ (root, str(uuid.uuid4())) for root in roots ])
```

Comment:
May I propose `make_unique_id()` to assert in all cases what a unique ID is and make future changes easier.

Reply:
Sure that seems reasonble